client/src/components/modals/incorporate-login.vue

```diff
@@ -1,26 +1,32 @@
 <template>
-  <v-dialog v-model="showModal" max-width="60%" >
+  <v-dialog v-model="showModal" max-width="850px" persistent>
     <v-card class="pa-4">
-      <sbc-authentication-options></sbc-authentication-options>
+      <sbc-authentication-options :redirectUrl="getNameRequestUrl" @close="toggleModal(false)"/>
     </v-card>
   </v-dialog>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
-import SbcAuthenticationOptions from 'sbc-common-components/src/components/SbcAuthenticationOptions.vue'
+import { mapGetters } from 'vuex'
 import newReqModule from '@/store/new-request-module'
+import SbcAuthenticationOptions from 'sbc-common-components/src/components/SbcAuthenticationOptions.vue'
 
 @Component({
   components: {
     SbcAuthenticationOptions
+  },
+  computed: {
+    ...mapGetters(['getNameRequestUrl'])
   }
 })
 export default class IncorporateLoginModal extends Vue {
+  readonly getNameRequestUrl!: string
+
   get showModal () {
     return newReqModule.incorporateLoginModalVisible
   }
-  set showModal (value: boolean) {
+  toggleModal (value: boolean) {
```

Comment:
It's not toggling if you set the value 😛

Reply:
Valid! fixed in next commit 😆